_drafts/conda.md

```diff
@@ -0,0 +1,698 @@
+has limited or no access to the Internet. These packaged environments can be
+used **without** installing Python, Conda, or any other packages on the host
+machine.
+
+## Creating Your Own Conda Packages
+
+There is a high chance that all packages required by your project, team, or
+developers are not yet built for Conda and hosted on the Conda Forge repository.
+Fret not, as building and publishing Conda packages is relatively simple. We'll
+quickly walk through how I built and published the `gcc-arm-none-eabi` package
+that is hosted [here](https://anaconda.org/tyhoff17/gcc-arm-none-eabi) and built
+from the official
+[ARM GNU Embedded Toolchain](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads).
+
+Creating a package for this toolchain is straight-forward since the binaries are
+already built for each platform. All we need to do is download the tarball,
+extract it, and copy the contents into a Conda package.
+
+> If you want to jump ahead and look at the final recipe, it is hosted in
+> Memfault's
+> [conda-recipes Github repo](https://github.com/memfault/conda-recipes/tree/master/gcc-arm-none-eabi).
+
+### Creating the Build Environment
+
+The first thing we need to do is create an environment which has `conda-build`
+and `anaconda-client` installed, both of which will be used to build and publish
+our package.
+
+```
+# Create the Conda Environment we'll use for building
+$ conda create -n build
+
+# Activate the environment called `build`
+$ conda activate build
+
+# Install the necessary packages to build and publish Conda packages
+(build)
+$ conda install conda-build anaconda-client
+```
+
+Now we'll want to create an empty directory, with a single `meta.yaml` file
+within it.
+
+```
+$ mkdir /tmp/arm-none-eabi-gcc && cd /tmp/arm-none-eabi-gcc
+$ touch meta.yaml
+```
+
+### Defining the `meta.yaml` File
+
+Next, we'll set up our `meta.yaml` file with the following information, with
+each field described below. The definition is small because we are merely
+repackaging binaries.
+
+```yaml
+package:
+  name: gcc-arm-none-eabi
+  version: 8.2019.q3.update
+
+source:
+  url: https://developer.arm.com/-/media/Files/downloads/gnu-rm/8-2019q3/RC1.1/gcc-arm-none-eabi-8-2019-q3-update-mac.tar.bz2 # [osx]
+  sha256: fc235ce853bf3bceba46eff4b95764c5935ca07fc4998762ef5e5b7d05f37085 # [osx]
+
+build:
+  string: "0"
+
+test:
+  commands:
+    - arm-none-eabi-gcc --version
+```
+
+- `package`: This is the Name and Version of the Conda package that we are
+  building. e.g. if we are building Make 4.2.1 for Conda, then Name would be
+  "Make" and Version would be "4.2.1".
+- `source`: This is either the fields `url` and `sha256` which points to a Zip
+  or Tarball containing the source code or binaries, or it can also reference a
+  Git repository using the `git_url` and `git_rev` fields.
+- `build`: This is the build number. If a user publishes a package but it was
```

Comment:
Can you add a note on the restriction for the `build` property? Does it need to be a monotonically increasing integer?